packages/hollywood.rb

```diff
@@ -23,20 +23,12 @@ class Hollywood < Package
   depends_on 'tree'
   depends_on 'util_linux'
 
-  def self.patch
-    system "sed -i 's:WIDGET_DIR=\"$(dirname $0)/../lib/$PKG\":WIDGET_DIR=\"$(dirname $0)/../libexec/$PKG\":' ./bin/hollywood"
-  end
-  
-  def self.build
-    FileUtils.cp_r "./lib", "./libexec"
-  end
-  
   def self.install
     FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"
-    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/libexec"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/lib"
     FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share/man/man1"
     FileUtils.cp "./bin/hollywood", "#{CREW_DEST_PREFIX}/bin/"
-    FileUtils.cp_r "./libexec/hollywood/", "#{CREW_DEST_PREFIX}/libexec/"
+    FileUtils.cp_r "./lib/hollywood/", "#{CREW_DEST_PREFIX}/lib/"
```

Comment:
Bump version?

Reply:
It's just a minor fix that won't affect any installs except mine since Hollywood isn't in the main repos yet. I don't think a version bump is necessary